apps/api/drizzle/0055_mute_purple_man.sql

```diff
@@ -0,0 +1,2 @@
+DROP INDEX "uq_rewards_competition_id_address";--> statement-breakpoint
+CREATE UNIQUE INDEX "uq_rewards_competition_id_user_id_agent_id" ON "rewards" USING btree ("competition_id","user_id","agent_id");
```

Comment:
I have to drop the constraint that we have on  `("competition_id","user_id")` Changing it to `("competition_id","user_id","agent_id")`. Meaning you can have more than one reward per competition per user per agent

note: if you are a booster, `agent_id` is null

Reply:
So for a single competition, a user can get rewards for multiple agents?